Makefile

```diff
@@ -275,6 +275,11 @@ BUILD_OUTPUT=/tmp
 
 release_github:	build-cli-darwin build-cli-linux create-gh-release upload-gh-darwin upload-gh-linux
 
+build-cli:
+	cargo build --bin fluvio
+
+build-cluster:
```

Comment:
Can you do re-factor this? 
```
build_test:	install_test_target
	cargo build $(TEST_RELEASE_FLAG) $(TEST_TARGET) --bin fluvio $(VERBOSE)
	cargo build $(TEST_RELEASE_FLAG) $(TEST_TARGET) --bin fluvio-run $(VERBOSE)
	cargo build $(TEST_RELEASE_FLAG) $(TEST_TARGET) --bin flv-test $(VERBOSE)
```

So it will be like this:
```
build_cluster:	install_test_target
	cargo build $(TEST_RELEASE_FLAG) $(TEST_TARGET) --bin fluvio $(VERBOSE)
	cargo build $(TEST_RELEASE_FLAG) $(TEST_TARGET) --bin fluvio-run $(VERBOSE)

build_test: build_cluster
       cargo build $(TEST_RELEASE_FLAG) $(TEST_TARGET) --bin flv-test $(VERBOSE)

```

Reply:
Let's hope tests pass 🤞

Reply:
ironically enough, making this change is where I introduced a build bug 🐛